StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/New Payment Method Screen/AddPaymentMethodViewController.swift

```diff
@@ -313,10 +300,15 @@ class AddPaymentMethodViewController: UIViewController {
     }
 
     private func updateFormElement() {
-        if selectedPaymentMethodType == .stripe(.USBankAccount),
-            let usBankAccountFormElement = usBankAccountFormElement
-        {
-            paymentMethodFormElement = usBankAccountFormElement
+        if selectedPaymentMethodType == .stripe(.USBankAccount) {
+            if let usBankAccountFormElement {
+                // Use the cached form instead of creating a new one
+                paymentMethodFormElement = usBankAccountFormElement
+            } else {
+                // Cache the form
+                paymentMethodFormElement = makeElement(for: .stripe(.USBankAccount))
+                usBankAccountFormElement = (paymentMethodFormElement as! USBankAccountPaymentMethodElement)
```

Comment:
I'm realizing that theoretically this _could_ crash.  So maybe it should just be `as?` instead, and not send an error here b/c it's FormFactory's responsibility?

Reply:
Changed it to `as?` as proposed. As for the metric, agree we don't need to metric here for the reason you mentioned, and because `handleCollectBankAccount()` will fire it's .unexpectedPaymentSheetError if it is not returning a `USBankAccountPaymentMethodElement`